src/Apps/Order/routes.tsx

```diff
@@ -0,0 +1,22 @@
+import { graphql } from "react-relay"
+import { OrderApp } from "./OrderApp"
+
+// @ts-ignore
+import { ComponentClass, StatelessComponent } from "react"
+
+export const routes = [
+  {
+    path: "/order2/:orderID",
+    Component: OrderApp,
+    query: graphql`
+      query routes_OrderQuery {
+        # order(id: $orderID) {
```

Comment:
We can faff with this as we integrate into force-- hopefully in the actual use-case it will be trivial (as long as we're passing in your access token, which we must be to get `me`, you should be able to fetch the order you just created)